### DISCUSSION TITLE
Is there a way to stop the camera from automatically starting shooting?

### DISCUSSION THREAD

**Discussion by matsumotokobe** [CommunityToolkit/Maui#2013] [Q&A]
Even if you try the following method at the time of loading, shooting will start.
 ・IsEnabled="False"
 ・this.cameraView.AutoStartPreview=false
・ this.cameraView.StopCameraAsync()

**Comment by GeorgeLeithead** [comment]
If you have a look at the documentation (or the samples app), the [Stop Preview](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/views/camera-view?tabs=windows#stop-preview) "provides the ability to programmatically stop the preview from the camera."
